ladybug/src/main/java/nl/nn/ibistesttool/LadybugPipe.java

```diff
@@ -61,15 +64,15 @@
  *
  */
 public class LadybugPipe extends FixedForwardPipe {
-	private static final Logger log = LogUtil.getLogger(LadybugPipe.class); // Overwrites log of JdbcFacade (using nl.nn.testtool.util.LogUtil instead of nl.nn.adapterframework.util.LogUtil)
+	private static final Logger log = LogUtil.getLogger(MethodHandles.lookup().lookupClass()); // Overwrite log of parent (using package nl.nn.ibistesttool for which logging will go to testtool4${ctx:instance.name.lc}.log)
```

Comment:
Je zegt hier: "Overwrite log of parent", maar dat is niet zo. Je creert hier een lokale logger. De logging van de parent gaat nog steeds naar dezelfde logger als eerst

Reply:
Ik heb even een klein testje gedaan en een log.info aan de config methode van AbstractPipe, FixedForwardPipe en LadybugPipe toegevoegd en dat geeft het volgende in testtool4webapp.log:

2020-12-14 12:11:32,039 INFO [main] nl.nn.ibistesttool.LadybugPipe Hello AbstractPipe
2020-12-14 12:11:32,039 INFO [main] nl.nn.ibistesttool.LadybugPipe Hello FixedForwardPipe
2020-12-14 12:11:32,039 INFO [main] nl.nn.ibistesttool.LadybugPipe Hello LadybugPipe

Ik zie dat ik de hele regel net zo goed weg kan laten, dat geeft hetzelfde resultaat (AbstractPipe gebruikt this in LogUtil.getLogger(this))

Reply:
Ik heb deze regel (en eenzelfde in Storage.java) weggehaald